CONTRIBUTOR-ROLES.md

```diff
@@ -0,0 +1,123 @@
+# Contributor Roles
+
+### Triage Team
+Triagers are members that help triage issues. They can set assignments and labels.
+
+* Responsibilities:
+    * Assigning and labeling issues 
+
+* Qualifications: 
+    * Must be a Cilium Organization member
+
+* Privileges:
+    * GitHub Triage role
+
+The process of becoming a Triage Team member is:
+
+1. The contributor is nominated by opening a PR against the member management file under the triage team
+2. At least two members who are already Committers, approve the PR
+
+### Security Team
+Security Team members are listed in the security team file. The security team is responsible for responding to all security issues reported to security@cilium.io or any other community forum. They ensure that the issues are mitigated and a responsible disclosure process is followed in a reasonable time.
+
+* Responsibilities:
+    * All reports are thoroughly investigated by the Security Team.
+    * Any vulnerability information shared with the Security Team is not shared with others unless necessary to fix the issue. Information is shared only on a need to know basis.
+    * As the security issue moves through the identification and resolution process, the reporter will be notified
+    * Security Team members must uphold the Security embargo policy
```

Comment:
Some bullets are full sentences with periods at the end, some are not. We should do a grammar pass and make these consistent (across both docs).